docs/README.md

```diff
@@ -30,4 +30,5 @@
     * [Why only traces in UI?](cn/FAQ/Why-have-traces-no-others.md)
     * [Too many GRPC logs in the console](cn/FAQ/Too-many-gRPC-logs.md)
     * [The trace doesn't continue in kafka consumer side](en/FAQ/kafka-plugin.md)
+    * [Protoc-Plugin-Fails-When-Build](en/FAQ/Protoc-Plugin-Fails-When-Build.md)
```

Comment:
Use English sentence, instead of file name.

Reply:
Ok, that's changed.